src/test/java/com/mathworks/ci/RunMatlabTestsBuilderTest.java

```diff
@@ -524,4 +524,135 @@ public void verifySystemTempDirDeleted() throws Exception {
         FreeStyleBuild build = project.scheduleBuild2(0).get();
         jenkins.assertLogContains("rmdir(tmpDir,'s')", build);
     }
+    
+    /*
+     * Test To verify if Logging level is set correctly 
+     * 
+     */
+
+    @Test
+    public void verifyLoggingLevelSet() throws Exception {
+        this.buildWrapper.setMatlabBuildWrapperContent(new MatlabBuildWrapperContent(
+                Message.getValue("matlab.custom.location"), getMatlabroot("R2018b")));
+        project.getBuildWrappersList().add(this.buildWrapper);
+        Map<String, String> logginggLevel = new HashMap<String, String>();

```

Comment:
Should `logginggLevel ` be `loggingLevel`? (Extra g)

Reply:
Done.